Sources/OpenAPIRuntime/Conversion/CurrencyExtensions.swift

```diff
@@ -177,12 +177,50 @@ extension Converter {
         try decoder.decode(T.self, from: data)
     }
 
+    func convertURLEncodedFormToCodable<T: Decodable>(
+        _ data: Data
+    ) throws -> T {
+
+        let decoder = URIDecoder(
+            configuration: .init(
+                style: .form,
+                explode: true,
+                spaceEscapingCharacter: .percentEncoded,
```

Comment:
```suggestion
                spaceEscapingCharacter: .plus,
```

According to https://www.rfc-editor.org/rfc/rfc1866.html#section-8.2.1

> 1. The form field names and values are escaped: space
        characters are replaced by `+', and then reserved characters
        are escaped as per [URL];